ext/scripts/tests/install_scripts/test_files/apt/with_versions/some_missing_versions

```diff
@@ -1,2 +1,3 @@
 curl|7.58.0-2ubuntu3.16 # t1
 wget| # t2
+git|
```

Comment:
Hm, the path is
```ext/scripts/tests/install_scripts/test_files/apt/with_versions/some_missing_versions ```
Shouldn't we declare the version then?

Reply:
this is one of the test files for the package scripts, I added git, because it has a colon in its version number